aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/adapter/SQLiteTable.java

```diff
@@ -173,6 +175,22 @@ public String getPrimaryKeyColumnName() {
         return Immutable.of(foreignKeys);
     }
 
+    /**
+     * Returns the SQLiteColumn for the provided columnName.
+     * @param columnName columnName of the column
```

Comment:
Nit: Instead of "columnName of the column" it may be more clear to say "name of the column"